packages/grafana-ui/src/options/builder/axis.tsx

```diff
@@ -38,7 +38,7 @@ export function addAxisConfig(
       },
       showIf: (c) => c.axisPlacement !== AxisPlacement.Hidden,
       // no matter what the field type is
```

Comment:
This comment ain't relevant anymore :)
```suggestion
      // Do not apply default settings to time and string fields which are used as x-axis fields in Time series and Bar chart panels
```